src/views/Home.jsx

```diff
@@ -21,6 +42,18 @@ export function Home({ makeNewList }) {
 			<button type="button" onClick={handleClick}>
 				Make new list
 			</button>
+			<form onSubmit={handleShareListSubmit}>
```

Comment:
I did get a little confused by references to "sharing" a list instead of "joining" one

Reply:
Updated the function naming to address this!